iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/plugin/KubernetesParser.java

```diff
@@ -63,4 +74,34 @@ public static boolean hasHelmContent(String text) {
     }
     return false;
   }
+
+  @Nullable
+  String evaluateHelmTemplate(String filename, String source, InputFileContext inputFileContext) {
+    // TODO: better support of Helm project structure
+    var valuesFile = findValuesFile(inputFileContext);
+    if (valuesFile != null) {
+      try {
+        return helmPreprocessor.evaluateTemplate(filename, source, valuesFile.contents());
```

Comment:
The `content` will be a better name than `source` here

Reply:
It's passed as `source` all the way from `TreeParser.parse`, and I believe the original template should still be considered source in our case